src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -203,4 +210,23 @@ protected ReadThreadingAssemblerArgumentCollection getReadThreadingAssemblerArgu
     @Advanced
     @Argument(fullName= USE_FILTERED_READS_FOR_ANNOTATIONS_LONG_NAME, doc = "Use the contamination-filtered read maps for the purposes of annotating variants", optional=true)
     public boolean useFilteredReadMapForAnnotations = false;
+
+    public String[] getDragenNameValuePairs() {
+        return new String[]{
+                APPLY_BQD_LONG_NAME, "true",
+                APPLY_FRD_LONG_NAME, "true",
+                TRANSFORM_DRAGEN_MAPPING_QUALITY_LONG_NAME, "true",
+                SOFT_CLIP_LOW_QUALITY_ENDS_LONG_NAME, "true",
+                MAPPING_QUALITY_THRESHOLD_FOR_GENOTYPING_LONG_NAME, "1",
+                ReadFilterArgumentDefinitions.MINIMUM_MAPPING_QUALITY_NAME, "1",
+                ALLELE_EXTENSION_LONG_NAME, "1",
+                LikelihoodEngineArgumentCollection.DISABLE_CAP_BASE_QUALITIES_TO_MAP_QUALITY_LONG_NAME, "true",
+                LikelihoodEngineArgumentCollection.ENABLE_DYNAMIC_READ_DISQUALIFICATION_FOR_GENOTYPING_LONG_NAME, "true",
+                LikelihoodEngineArgumentCollection.EXPECTED_MISMATCH_RATE_FOR_READ_DISQUALIFICATION_LONG_NAME, "0.03",
+                GenotypeCalculationArgumentCollection.GENOTYPE_ASSIGNMENT_METHOD_LONG_NAME, String.valueOf(GenotypeAssignmentMethod.USE_POSTERIOR_PROBABILITIES),
+                AssemblyRegionArgumentCollection.INDEL_PADDING_LONG_NAME, "150",
+                GenotypeCalculationArgumentCollection.CALL_CONFIDENCE_LONG_NAME, "3.0",
+                GenotypeCalculationArgumentCollection.USE_POSTERIORS_TO_CALCULATE_QUAL_LONG_NAME, "true",
+        };
+    }
```

Comment:
What about `hcArgs.informativeReadOverlapMargin = 1;` from the old settings?

Also, were `ReadFilterArgumentDefinitions.MINIMUM_MAPPING_QUALITY_NAME, "1"` and `ALLELE_EXTENSION_LONG_NAME, "1"` set using the old method? I don't see them

Reply:
`hcArgs.informativeReadOverlapMargin = 1;` is handled. That is the `ALLELE_EXTENSION_LONG_NAME, "1"` in that list. Its argument name and its argument collection variable are fairly confusingly discordant. 

`MAPPING_QUALITY_THRESHOLD_FOR_GENOTYPING_LONG_NAME, "1",` is also handled (which is distinct from the mapping quality on the read filter but that is handled  by `HaplotypeCaller.customCommandLineValidaiton()`